src/components/basics/GatsbyLink.stories.js

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 import GatsbyLink from './GatsbyLink';
 
-storiesOf('Basics|GatsbyLink', module)
-  .addParameters({ component: GatsbyLink })
-  .add('default', () => <GatsbyLink to="/">I am a GatsbyLink</GatsbyLink>);
+export default {
+  component: GatsbyLink,
+  title: 'Basics/GatsbyLink',
+};
+
+export const Basic = () => <GatsbyLink to="/">I am a GatsbyLink</GatsbyLink>;
```

Comment:
```suggestion
export const Default = () => <GatsbyLink to="/">I am a GatsbyLink</GatsbyLink>;
```

Suggest don't use `storyName` when not needed for cleanliness

Reply:
This was fixed in a prior commit. Thanks for the time reviewing it.